services/app-api/handlers/rate/shared/calculateAndPutRate.ts

```diff
@@ -37,17 +48,20 @@ export const calculateAndPutRate = async (
     const data = await getMeasureByCoreSet(combinedCoreSet!, pathParameters);
     const formattedData = formatMeasureData(data);
 
+    //check which data is valid for summation
+    const validatedData = validateMeasureData(formattedData);
```

Comment:
Suggestion: to match the function rename above

```suggestion
    const validatedData = removeEmptyRates(formattedData);
```